libcloud/test/dns/fixtures/cloudflare/nul.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@Kami nul is a reserved file name in Windows, so anyone trying to test or use this on Windows is going to have a lot a bad time.. Please can you rename it